### PR TITLE
add function to create new info pages

### LINKED ISSUE #34: Autosave for protocols
The protocol editor should auto save content in case our beloved computer crashes.

### PR DESCRIPTION
**Pull request by Nef10**
fix #34 you can restore autosaves of new sites

### CODE REVIEW

**Comment by invliD** [comment]
Please rebase once #109 has been merged.

File: _1327/information_pages/views.py [hunk 1]
```diff
@@ -14,6 +16,11 @@
 from _1327.information_pages.models import InformationDocument
 from _1327.user_management.shortcuts import get_object_or_error
 
+def create(request):
+	title = "New Page from "+str(datetime.now());
```
Comment (invliD) [inline _1327/information_pages/views.py]: Make this translatable.

File: _1327/main/context_processors.py [hunk 2]
```diff
@@ -31,3 +31,6 @@ def mark_selected(request, menu_item):
 	if current_view == item_view:
 		menu_item.selected = True
 		return True
+
+def can_create_informationpage(request):
+	return {'CAN_CREATE_INFORMATIONPAGE': request.user.has_perm("information_pages.add_informationdocument")}
```
Comment (invliD) [inline _1327/main/context_processors.py]: Modify this to a more general "Create document" variant, see below.

File: _1327/templates/base.html [hunk 3]
```diff
@@ -48,6 +48,13 @@
 						{% endif %}
 					</li>
 					{% endfor %}
+					{% if CAN_CREATE_INFORMATIONPAGE %}
+						<li class="pull-right">
+							<a href="{% url 'information_pages:create' %}">
+								<span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
+							</a>
+						</li>
+					{% endif %}
```
Comment (invliD) [inline _1327/templates/base.html]: We can't create such a link for each type of document, see above.
What about one link if the user can create *any* type of document?

Comment (janno42) [inline _1327/templates/base.html]: i would say it's correct to have this here for info pages only. the only other types planned for now are transcripts and polls. both of them should be somehow collected in a list on specific pages (a list of transcripts and a list of polls) and the option to add a new page of this type can then be added on these pages.

Comment (invliD) [inline _1327/templates/base.html]: Agreed.

**Comment by invliD** [comment]
:+1:

**Pull request merged**